packages/react/src/Avatar/Avatar.module.css

```diff
@@ -0,0 +1,114 @@
+.Avatar {
+  position: relative;
+  display: inline-block;
+  overflow: hidden;
+  width: var(--base-size-40);
+  height: var(--base-size-40);
+}
+
+.Avatar--shape-circle {
+  border-radius: var(--brand-borderRadius-full);
+}
+
+.Avatar--shape-square {
+  border-radius: var(--brand-borderRadius-medium);
+}
+
+.Avatar-iamge {
```

Comment:
```suggestion
.Avatar-image
```

Reply:
Not sure if this is actually being used though? Can't see the corresponding call for it.

Reply:
Good catch! These styles were meant to handle possible non-square images. I've fixed the class name and updated the component to use it properly.